deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -225,15 +239,53 @@ public void run() {
 			LOG.info("Starting deposit workers");
 			cdrMetsDepositWorkerPool.run();
 		}
-
-		// Repopulate the queue
-		requeueAll();
+	}
+	
+	private Map<String, Set<String>> getQueuedDepositsWithJobs() {
+		Map<String, Set<String>> depositMap = new HashMap<>();
+		addQueuedDeposits(
+				RedisWorkerConstants.RESQUE_QUEUE_PREFIX + RedisWorkerConstants.DEPOSIT_PREPARE_QUEUE, depositMap);
+		addQueuedDeposits(
+				RedisWorkerConstants.RESQUE_QUEUE_PREFIX + RedisWorkerConstants.DEPOSIT_DELAYED_QUEUE, depositMap);
+		addQueuedDeposits(
+				RedisWorkerConstants.RESQUE_QUEUE_PREFIX + RedisWorkerConstants.DEPOSIT_CDRMETS_QUEUE, depositMap);
+		return depositMap;
+	}
+	
+	private void addQueuedDeposits(String queueName, Map<String, Set<String>> depositMap) {
```

Comment:
Nothing in the Client class, but it looks like jesque web is polling jobs using this:

https://github.com/gresrun/jesque/blob/master/src/main/java/net/greghaines/jesque/meta/dao/impl/QueueInfoDAORedisImpl.java#L152